docker-compose.test.yml

```diff
@@ -13,6 +13,7 @@ services:
         KERAS_PACKAGE: keras==2.4.3
         PYTORCH_PACKAGE: torch==1.8.1+cpu
         TORCHVISION_PACKAGE: torchvision==0.9.1+cpu
+        PYTORCH_LIGHTNING: pytorch_lightning==1.2.6
```

Comment:
Can you move `PYTORCH_LIGHTNING` below `PYTORCH` here as well?

Reply:
sure